src/extras/module-types.js

```diff
@@ -33,6 +35,9 @@
       if (cssContentType.test(contentType))
         return res.text()
         .then(function (source) {
+          source = source.replace(/url\(\s*(["'])?((?:\\.|[^"')\\])+)\1\s*\)/g, function (match, quotes, relUrl) {
```

Comment:
Per your first spec comment I guess we should include `,` and `(` as well:

```suggestion
          source = source.replace(/url\(\s*(["'])?((?:\\.|[^,"'()\\])+)\1\s*\)/g, function (match, quotes, relUrl) {
```

Reply:
How about whitespace characters?

Reply:
Ah of course, added.

Reply:
That looks great.